spec/models/social_networking/profile_answer_spec.rb

```diff
@@ -0,0 +1,21 @@
+require "spec_helper"
+
+module SocialNetworking
+  describe ProfileAnswer, type: :model do
```

Comment:
My thought was to make sure we had a validation and not just a psql db integrity check.  Do you think I am essentially testing rails and thus the test is unnecessary?